src/services/settingsService.js

```diff
@@ -74,10 +74,16 @@ const setPlaySounds = async function(hasUserAccount, value) {
 	}
 }
 
+const setMediaDefaultState = async function(value) {
+	await axios.post(generateOcsUrl('apps/provisioning_api/api/v1/config/users/spreed/calls_start_without_media'),
```

Comment:
we are going to reuse this heavily going forward. I would recommend to make a function that receives the configKey and configValue as arguments :)